diff --git a/experimental/clashapi/connections.go b/experimental/clashapi/connections.go
--- a/experimental/clashapi/connections.go
+++ b/experimental/clashapi/connections.go
@@ -86,6 +86,7 @@ func (c connectionObject) MarshalJSON() ([]byte, error) {
 		"metadata": map[string]any{
 			"network":         c.Metadata.Network,
 			"type":            inbound,
+			"inboundUser":     c.Metadata.User,
 			"sourceIP":        c.Metadata.Source.Addr,
 			"destinationIP":   c.Metadata.Destination.Addr,
 			"sourcePort":      F.ToString(c.Metadata.Source.Port),
